airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostresSourceDatatypeTest.java

```diff
@@ -54,13 +54,13 @@ protected Database setupDatabase() throws SQLException {
         "org.postgresql.Driver",
         SQLDialect.POSTGRES);
 
-    database.query(ctx -> ctx.fetch("CREATE SCHEMA TEST;"));
-    database.query(ctx -> ctx.fetch("CREATE TYPE mood AS ENUM ('sad', 'ok', 'happy');"));
-    database.query(ctx -> ctx.fetch("CREATE TYPE inventory_item AS (\n"
-        + "    name            text,\n"
-        + "    supplier_id     integer,\n"
-        + "    price           numeric\n"
-        + ");"));
+    database.query(ctx -> {
+      ctx.execute("CREATE SCHEMA TEST;");
+      ctx.execute("CREATE TYPE mood AS ENUM ('sad', 'ok', 'happy');");
+      ctx.execute("CREATE TYPE inventory_item AS (name text, supplier_id integer, price numeric);");
+      ctx.execute("SET lc_monetary TO 'en_US.utf8';");
```

Comment:
should the actual source connector also run this query? I don't follow why we run tests against this configuration -- do we expect all user databases to also have this configuration?

Reply:
The purpose of this setting is to make sure the locale is US, and the currency symbol is `$`, which matches what we expect in the unit test case. User databases do not need to have this config. They can set it to any locale, but the output currency symbol will be different.

But I will remove this, since I agree that we'd better convert the money value to double. So the currency symbol is no loner a concern.